core/src/operations/mod.rs

```diff
@@ -107,13 +107,13 @@ pub(super) mod opcode_constants {
     pub const OPCODE_SPAN: u8       = 0b0101_0110;
     pub const OPCODE_JOIN: u8       = 0b0101_0111;
     pub const OPCODE_DYN: u8        = 0b0101_1000;
-    pub const OPCODE_RCOMBBASE: u8  = 0b0101_1001;
+    pub const OPCODE_HORNEREXT: u8  = 0b0101_1001;
     pub const OPCODE_EMIT: u8       = 0b0101_1010;
     pub const OPCODE_PUSH: u8       = 0b0101_1011;
     pub const OPCODE_DYNCALL: u8    = 0b0101_1100;
 
     pub const OPCODE_MRUPDATE: u8   = 0b0110_0000;
-    /* unused:                        0b0110_0100 */
+    pub const OPCODE_HORNERBASE: u8 = 0b0110_0100;
```

Comment:
I'm assuming we put Horner base here because the constraints require very high degree?

Reply:
That's the rational